src/index.js

```diff
@@ -6,7 +6,6 @@ import register from './middlewares/register.app';
 import { sequelize } from './api/models/index';
 import environnements from './configs/environments';
 
```

Comment:
@Ramadhan0 please, remove this file from your changes as it's not related to your task.